sw/device/tests/aes_masking_off_test.c

```diff
@@ -0,0 +1,140 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "hw/ip/aes/model/aes_modes.h"
```

Comment:
Is this exported by a Bazel rule? You may have to add a library to capture this dependency.

Reply:
ack. SG. I was expecting the build to fail hehe. If it didn't fail it must mean that there is a dependency exporting this somehow.